core/domain/collection_services.py

```diff
@@ -928,39 +928,28 @@ def update_collection(
     collection = apply_change_list(collection_id, change_list)
 
     _save_collection(committer_id, collection, commit_message, change_list)
-    update_collection_summary(collection.id, committer_id)
+    regenerate_collection_summary(collection.id, committer_id)
 
     if (not rights_manager.is_collection_private(collection.id) and
             committer_id != feconf.MIGRATION_BOT_USER_ID):
         user_services.update_first_contribution_msec_if_not_set(
             committer_id, utils.get_current_time_in_millisecs())
 
 
-def create_collection_summary(collection_id, contributor_id_to_add):
+def regenerate_collection_summary(collection_id, contributor_id_to_add):
     """Creates and stores a summary of the given collection.
```

Comment:
Update the docstring as per the new function name*

Reply:
Will be done in next commit.

Reply:
Done.